survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/question_parts/MultipleChoicePart.kt

```diff
@@ -43,7 +44,7 @@ internal class MultipleChoicePart @JvmOverloads constructor(
         }
 
     @ColorInt
-    var defaultColor: Int = ContextCompat.getColor(context, R.color.survey_text)
+    var defaultColor: Int = ContextCompat.getColor(context, R.color.question_text)
```

Comment:
Why changing the name?

Reply:
Not quite sure if we need such a thing. String resources are there as a source of truth and if you already have the element, I see no reason in making another element that only points to it. @KlausNie any thoughts?